opencga-server/src/main/java/org/opencb/opencga/server/rest/admin/AdminWSServer.java

```diff
@@ -324,23 +334,6 @@ public Response install(
 //        return createErrorResponse(new NotImplementedException());
 //    }
 
-    @POST
```

Comment:
Isn't part of the solution to call this endpoint after getting the list of organizations?